vhdbuilder/packer/create-managed-disk-from-sig-gen2.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash -e
+
+required_env_vars=(
+    "subscription_id"
+    "resource_group_name"
+    "create_time"
+    "location"
+    "os_type"
+)
+
+
+for v in "${required_env_vars[@]}"
+do
+    if [ -z "${!v}" ]; then
+        echo "$v was not set!"
+        exit 1
+    fi
+done
+
+sig_resource_id="/subscriptions/${subscription_id}/resourceGroups/${resource_group_name}/providers/Microsoft.Compute/galleries/PackerSigGallery/images/1804Gen2/versions/1.0.${create_time}"
+disk_resource_id="/subscriptions/${subscription_id}/resourceGroups/${resource_group_name}/providers/Microsoft.Compute/disks/1.0.${create_time}"
```

Comment:
Does Packer for Gen2 support how many replicacount, specifying regions for sig publish

Reply:
again, this is not the final SIG. I intentionally want to separate the build and release of SIG in two pipelines so we have less garbage in our prod SIG. Will see how it goes though.